connexion/utils.py

```diff
@@ -131,9 +131,9 @@ def boolean(s):
     >>> boolean('false')
     False
     '''
-    if s == 'true':
+    if s.lower() == 'true':
```

Comment:
IMHO we should allow passing None values, i.e. I would check for None to make sure we always get proper ValueError (and not AttributeError).